### PR TITLE
Added support for getting total number of entrants in the draw

### PR DESCRIPTION
**Pull request by LuceCarter**

### CODE REVIEW

File: src/events/messages.ts [hunk 1]
```diff
@@ -179,6 +179,14 @@ tmi.on(
           tmi.say(config.channel, Giveaway.getNoEntrantsMessage());
         }
       }
+
+      if (message === Giveaway.commands.entrants) {
+        if(Giveaway.isOpen) {
+          tmi.say(config.channel, Giveaway.getCurrentEntrantsMessage())
+        } else {
+          tmi.say(config.channel, Giveaway.getNoEntrantsMessage());
```
Comment (whitep4nth3r) [inline src/events/messages.ts:187]: I wonder whether this should be changed to the access denied message?

Comment (whitep4nth3r) [reply src/events/messages.ts:187]: getInactiveMessage()?

File: src/events/messages.ts [hunk 2]
```diff
@@ -182,9 +182,15 @@ tmi.on(
 
       if (message === Giveaway.commands.entrants) {
         if(Giveaway.isOpen) {
-          tmi.say(config.channel, Giveaway.getCurrentEntrantsMessage())
+          if(Giveaway.entrants.size <= 0)
```
Comment (LuceCarter) [inline src/events/messages.ts:185]: No idea whats going on here with the line indentation, I did it in a hurry lol

Comment (whitep4nth3r) [reply src/events/messages.ts:185]: Please allow mods to use this command as discussed :)

**Comment by whitep4nth3r** [comment]
Closing this as it is so old :)

**Pull request closed without merging**